modules/ROOT/pages/configure-client-management-openid-task.adoc

```diff
@@ -9,8 +9,7 @@ MuleSoft verifies support in Anypoint Platform for Salesforce, Okta, and OpenAM
 
 [NOTE]
 ====
-* Updates of clients created through this integration are not currently supported.
-* Deletion of clients created through this integration are supported with the limitation that the client is removed from Anypoint Platform but will still exist in the OpenID Connect Authorization Server, and API policies will still consider them valid. +
+Deletion of clients created through this integration is supported with the limitation that the client is removed from Anypoint Platform but will still exist in the OpenID Connect Authorization Server and API policies will still consider them valid until the cached token expires. +
```

Comment:
I don't think this is correct. If the client still exists on the provider, it will keep on being valid regardless any caching. Am I wrong Seb? @sebastiankorol ?

Reply:
As far as Auth Server is concerned the client doesn't exist. I'm unsure how a Gateway the openid connect policy would behave. When calling the OP introspect endpoint the response will be valid, but do you evict  your cache or otherwise care that the client is no longer in Auth Server?

Reply:
Okey, validated this with Gateway team. We can leave deletion text as it is changed on this PR.